.github/workflows/pull-request.yml

```diff
@@ -377,27 +397,26 @@ jobs:
       #     path: "./coverage/vitest/lcov.info"
       #     min_coverage: 86.6
 
+
   Test-Docusaurus-Deployment:
```

Comment:
1. Make no changes to this section. 
2. We are moving to PNPM for all documentation websites